src/Ng/CommandHelpers.cs

```diff
@@ -178,7 +178,8 @@ public static StorageFactory CreateSuffixedStorageFactory(string suffix, IDictio
                 { Arguments.StorageAccountName, Arguments.StorageAccountName + suffix },
                 { Arguments.StorageKeyValue, Arguments.StorageKeyValue + suffix },
                 { Arguments.StorageContainer, Arguments.StorageContainer + suffix },
-                { Arguments.StoragePath, Arguments.StoragePath + suffix }
+                { Arguments.StoragePath, Arguments.StoragePath + suffix },
+                { Arguments.StorageSuffix, Arguments.StorageSuffix + suffix }
```

Comment:
IMO we should rename `Arguments.StorageSuffix` to `Arguments.StorageEndpointSuffix`.

Having an argument named `StorageSuffix` is confusing given that we have a method named `CreateSuffixedStorageFactory`.